cmake_modules/FindProtobuf.cmake

```diff
@@ -26,6 +26,24 @@
 # PROTOBUF_EXECUTABLE: location of protoc
 
 
+find_package(Protobuf CONFIG)
+if(Protobuf_FOUND)
```

Comment:
Thank you for making a PR. Could you follow the existing coding style?

https://github.com/apache/orc/blob/ec2ea9c6aff8b8515452df651f08695639c18cbb/cmake_modules/FindProtobuf.cmake#L74

Reply:
I've updated the commit.